solutions/bubble_sort.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+# -- coding: utf-8 --
+"""
+A module for performing bubble sort on an array.
+
+Bubble sort is a simple sorting algorithm that repeatedly steps through the list,
+compares adjacent elements, and swaps them if they are in the wrong order. The
+pass through the list is repeated until the list is sorted.
+
+Module contents:
+    - bubble_sort(arr): Sorts an array in ascending order using the bubble sort algorithm.
+
+Created on 2025-1-4
+@author: Tomas Teclehaimanot
+"""
+
+
+def bubble_sort(arr):
```

Comment:
For Clarity:Add type annotations to specify that arr must be a list containing elements of type int or float, and that the function does not return anything (None).
```suggestion
def bubble_sort(arr: list[float | int]) -> None:
```

Reply:
Can you check again I have already updated it.

Reply:
done

Reply:
Great work